### PR TITLE
replace bootstrap with bootstrap vue

### LINKED ISSUE #1035: Generated Vue Code: Replace Boostrap with Boostrap-Vue
The following Bootstrap should be replaced by Bootstrap-Vue Components: 
- [x] Breadcrumb
- [x] Layout: Grid use stacked to horizontal BV layout
- [x] NavBar
- [x] Image
 

This is one sub point from #978

### PR DESCRIPTION
**Pull request by Tanya0609**
This is to address one of the Vue feedback: using Bootstrap-Vue. 

The following Bootstrap should be replaced by Bootstrap-Vue Components: 
- [x] [Breadcrumb](https://bootstrap-vue.js.org/docs/components/breadcrumb)

- [x] [Layout](https://bootstrap-vue.js.org/docs/components/layout): Grid should use _stacked to horizontal_ Bootstrap-Vue layout
- [x] [NavBar](https://bootstrap-vue.js.org/docs/components/navbar)

How to test:
1. you are able to generate a Vue project with 4 pages selected, the project should be able to start
2. the generated code should have changes contained from this PR. More details of Bootstrap Vue can be found in the links.
 
close #1035

### CODE REVIEW

File: templates/Web/Pages/Vue.Grid/src/components/GridComponent.vue [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ﻿<template>
-  <div class="col-md-4 col-sm-12 p-5">
+  <b-col sm="12" md="4" class="p-5">
     <img src="@/assets/GreyBox.svg" alt="Default Grey Box" class="mb-3" />
     <h3>{{header}}</h3>
     <p>{{description}}</p>
-  </div>
+  </b-col>
```
Comment (streetcoder123) [inline templates/Web/Pages/Vue.Grid/src/components/GridComponent.vue:6]: Just out of curiosity, what does `b-col` signify here?

File: templates/Web/Pages/Vue.MasterDetail/src/components/MasterDetailPage.vue [hunk 2]
```diff
@@ -7,14 +7,10 @@
...
-              <a class="breadCrumbLink" href="#">Master Detail</a>
-            </li>
-            <li class="breadcrumb-item active" aria-current="page">{{textSampleData.title}}</li>
-          </ol>
-        </nav>
+        <b-breadcrumb class="bg-white">
+          <b-breadcrumb-item href="#">Master Detail</b-breadcrumb-item>
+          <b-breadcrumb-item active>{{textSampleData.title}}</b-breadcrumb-item>
+        </b-breadcrumb>
       </div>
```
Comment (streetcoder123) [inline templates/Web/Pages/Vue.MasterDetail/src/components/MasterDetailPage.vue:14]: Nice.. much more concise ;)

File: templates/Web/Pages/Vue.Grid/src/components/GridComponent.vue (hunk 1 above)
Comment (Tanya0609) [reply templates/Web/Pages/Vue.Grid/src/components/GridComponent.vue:6]: Here I am wrapping a grid component (individual grid on the grid page) into `<b-col>`. It is Bootstrap-Vue style tag for grid layout. Here is a screenshot of the example provided by Bootstrap-Vue Docs.
Screen Shot 2019-08-07 at 8 11 36 PM

**Review by SahilTara — APPROVED** [review]

**Review by streetcoder123 — APPROVED** [review]

**Review by jcoc611-microsoft — APPROVED** [review]
Awesome

**Pull request merged**